test/blobtype_test.js

```diff
@@ -109,4 +109,14 @@ helper.contextualSuites('Entity (BlobType)', helper.makeContexts(Item),
         });
       });
     });
+
+    test('too-large blob (512k)', function() {
+      var id  = slugid.v4();7;
```

Comment:
`7;`

Reply:
I could blame the cat, or vim..

Reply:
Oh, that wasn't me!  Copy-pasta from the previous test case.  I can blame [jonas](dc2ef4ef55993d9162656c9b5f9b58517ecfcd8a) :)